src/parser.js

```diff
@@ -70,13 +70,8 @@ define([
             title = head.children('h\\:title, title'),
             binds = head.find('bind'),
             instances = _getInstances(xml),
-            data = $(instances[0]).children();
-
-        xml.find('setvalue').each(function () {
-            var $this = $(this);
-            form.addSetValue(
-                $this.attr('event'), $this.attr('ref'), $this.attr('value'));
-        });
+            data = $(instances[0]).children(),
+            setValues = xml.find('setvalue');
```

Comment:
I know you have not changed it here, but this `find` always felt way too liberal to me. Do you think this might be a reasonable time to change it to, for example, only find setvalue nodes that are immediate children of the `<model>` node? The same applies to finding `bind` nodes above. It seems that these two would incorrectly parse the form if anyone ever had a question ID of `bind` or `setvalue`.

Actually, let's not change that now. No need to possibly introduce more instability in a large-ish changeset like this. It's a good thing to keep in mind though. Curious to hear what @ctsims thinks too.